i18n/en.json

```diff
@@ -3529,6 +3529,5 @@
   "widgets.users_emails_input.loading": "Loading",
   "widgets.users_emails_input.no_user_found_matching": "No one found matching **{text}**, type email address",
   "widgets.users_emails_input.valid_email": "Add **{email}**",
-  "yourcomputer": "Your computer",
-  "youtube_video.notFound": "Video not found"
```

Comment:
Was curious on this one so I tested with youtube link with available and unavailable video (only by changing the last character):
- post this (available) link --> https://www.youtube.com/watch?v=EW9rvhRnP04
- then (not available) link --> https://www.youtube.com/watch?v=EW9rvhRnP05

On master/5.14, second post showed unavailable thumbnail placeholder from youtube (probably expected but not entirely sure as I was expecting a message saying "Video not found")
![Screen Shot 2019-08-22 at 6 14 31 PM](https://user-images.githubusercontent.com/5334504/63507017-57cde680-c509-11e9-9cb4-3eec81c26c3d.png)

On this PR, thumbnails shown as if both videos are available (clicking on second post reveals that the video is unavailable)
![Screen Shot 2019-08-22 at 6 16 03 PM](https://user-images.githubusercontent.com/5334504/63507065-7502b500-c509-11e9-956a-e8dc3397fba9.png)

Reply:
Interesting. The thumbnail populated is from opengraph data so it must have returned so, i don't think this would be a bug we should fix here in webapp. In any case let me check and file a ticket for the bug

Reply:
@saturninoabril Seems related to how opengraph provides data. Weird it happens for that link. I tried couple of other links and if it is not a valid youtube link all you see is the link as a message as there will be no metadata.

@hmhealey Quick thoughts? Should we be creating a ticket here?